ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -489,3 +489,6 @@ SMS_PREFIXES_BY_COUNTRY = '{{ localsettings.SMS_PREFIXES_BY_COUNTRY }}'
 # See corehq/apps/builds/README.md for more information.
 BASE_ADDRESS = '{{ SITE_HOST }}'
 EMAIL_USE_TLS = {{ localsettings.EMAIL_USE_TLS }}
+
+BANK_ROUTING_NUMBER_ACH = '{{ localsettings.BANK_ROUTING_NUMBER_ACH }}'
+BANK_ROUTING_NUMBER_WIRE = '{{ localsettings.BANK_ROUTING_NUMBER_WIRE }}'
```

Comment:
fyi @nickpell @benrudolph  if you add an environment variable like this you need to make sure it exists in all environments (this was missing from india) or ansible will choke on it. you can do a more protective style like #70 if you don't want to add it everywhere